app/components/projects-carousel.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+    Carousel,
+    CarouselContent,
+    CarouselDots,
+    CarouselItem,
+    CarouselNext,
+    CarouselPrevious,
+} from "../components/ui/carousel"
+import Project from "./project"
+
+interface ProjectsCarouselProps {
+    projects: { image?: string; title?: string; subtitle?: string; description?: string; link?: string; }[];
+}
+
+const ProjectsCarousel = ({
+    projects
+}: ProjectsCarouselProps) => {
+    return (
+        <div className='flex justify-center mt-4'>
+            <Carousel className="w-full max-w-3xl">
+                <CarouselContent className="-ml-2 md:-ml-4">
+                    {projects.map((project, index) => (<CarouselItem key={index} className="basis-1/3 pl-2 md:pl-4"><Project
```

Comment:
nit: stylistic change
```suggestion
                    {projects.map((project, index) => (
                        <CarouselItem key={index} className="basis-1/3 pl-2 md:pl-4">
                            <Project
```

Reply:
actually you can ignore these, the prettier change I merged should catch this